component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -346,6 +346,79 @@ public Response getUsers(@Context UriInfo uriInfo,
     return EntityBuilder.getResponse(collectionUser, uriInfo, RestUtils.getJsonMediaType(), Response.Status.OK);
   }
 
+  @POST
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/advancedfilter")
+  @RolesAllowed("users")
+  @Operation(
+      summary = "Gets all users",
+      method = "POST",
+      description = "")
+  @ApiResponses(value = {
+      @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+      @ApiResponse (responseCode = "404", description = "Resource not found"),
+      @ApiResponse (responseCode = "500", description = "Internal server error due to data encoding"),
+      @ApiResponse (responseCode = "400", description = "Invalid query input") })
+  public Response getUsersAllByAdvancedFilter(@Context UriInfo uriInfo,
```

Comment:
we should see if we should use separated endpoints for all users and connections

Reply:
I Think no need to repeat the some code to create tow endpoints